Cubical/Categories/Presheaf/Properties.agda

```diff
@@ -0,0 +1,393 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+
+module Cubical.Categories.Presheaf.Properties where
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.NaturalTransformation
+open import Cubical.Categories.Instances.SetCat
+open import Cubical.Categories.Instances.FunctorCat
+open import Cubical.Categories.Functor
+open import Cubical.Categories.Presheaf.Base
+open import Cubical.Categories.Equivalence
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Equiv using (fiber)
+open import Cubical.Data.Sigma
+
+import Cubical.Categories.Morphism as Morphism
+import Cubical.Categories.Constructions.Slice as Slice
+import Cubical.Categories.Constructions.Elements as Elements
+
+private
+  variable
+    ℓ ℓ' : Level
+
+-- TODO: upstream
```

Comment:
Yeah upstream somewhere. The name should probably reflect that this is for sets

Reply:
I'm not super familiar with the layout of the rest of this repo yet. Where should I upstream this to?

Reply:
I'm not sure... Maybe @ecavallo  can help me?

Reply:
Looks like some fiber-related results are collected in `Cubical.Functions.Fibration`?